site/src/sphinx/client-circuit-breaker.rst

```diff
@@ -1,6 +1,270 @@
+.. _CircuitBreaker by Martin Fowler: https://martinfowler.com/bliki/CircuitBreaker.html
+.. _LINE Engineering blog: https://engineering.linecorp.com/en/blog/detail/76
+.. _decorator: client-decorator.html
+
 .. _client-circuit-breaker:
 
-Circuit breakers
-================
+Circuit breaker
+===============
+
+In Microservice Architecture, it's common that various services running on different machines are connected to
+each other through remote calls. If one of the services unreachable somehow such as TCP hang, it will take long
```

Comment:
If one of the services unreachable -> If one of the services **becomes** unreachable?